spec/components/attachment_spec.rb

```diff
@@ -41,6 +40,16 @@ def component_name
     expect(rendered).to match(/2 pages/)
   end
 
+  it "can show file type that doesn't have an abbreviation" do
+    render_component(
+      attachment: {
+        title: "Attachment",
+        url: "attachment",
+        content_type: "text/plain",
+      },
+    )
+    expect(rendered).to match(/Plain Text/)
+  end
```

Comment:
Missing a new line here :-).

Reply:
Also, what do you think about a test for an unknown content type?

Reply:
Thanks re new line.

I agree it'd be good to test unknown but as there's nothing to output there's not a clear assertion to make. I'll see if I can test that with nothing to show it doesn't show a metadata paragraph (which it currently shows and probably shouldn't)

Reply:
I was wondering about this - are we wanting to couple the file types we support in our app to the ones govspeak can render? If so, then perhaps it should throw an exception for something unexpected; if not, then maybe we should consider 'Unknown type' or something?

Reply:
I'd suggest they're only loosely coupled - since this should gradually reflect all GOV.UK attachments. It may well transpire that this reflects more content types than Content Publisher supports. Due to the loose coupling I don't think exception would be the right route for an unrecognised content_type.

I think saying "Unknown type" wouldn't provide much value to a user, so I think the right course would be to not show anything. That'd be consistent with just omitting content_type, which is supported.